Tanvir04Khan/01-max-udemy/03-login-page/src/App.js

```diff
@@ -1,17 +1,27 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 
-import Header from "./Components/Login/Header";
+import LoginPage from "./Components/Login/LoginPage";
+import HomePage from "./Components/Home/HomePage";
 
 import "./App.css";
 
 function App() {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
 
+  useEffect(() => {
+    if (localStorage.isLoggedIn === "1") {
+      setIsLoggedIn(true);
+    }
+  }, []);
+
+  const isloggedin = (boolean) => {
+    setIsLoggedIn(boolean);
+  };
+
   return (
     <div className="App">
-      <Header />
-      <h1>Helllo There!</h1>
-      {}
+      {!isLoggedIn && <LoginPage isLoggedin={isloggedin} />}
+      {isLoggedIn && <HomePage isLoggedin={isloggedin} />}
```

Comment:
why u need to pass prop here ?

Reply:
you can use ternary operator here

Reply:
changed code as per suggestion @fayez-baig